README.md

```diff
@@ -38,6 +38,20 @@ That's it 🎉
 The following [example project](example/llrt-sam/) sets up a lambda
 instrumented with a layer containing the llrt runtime.
 
+### Option 4: AWS CDK
+
+You can use [`cdk-lambda-llrt` construct library](https://github.com/tmokmss/cdk-lambda-llrt) to easily deploy LLRT Lambda functions with AWS CDK.
```

Comment:
Nit: Remove "easy", we try to avoid it because it's very contextual :)

Reply:
Thanks, fixed!